backend/tests/unit/test_user_query.py

```diff
@@ -0,0 +1,32 @@
+from app.graphql.user import UserQuery
+import graphene
+from graphene.test import Client
+from app.resources.user_dto import UserDTO
+
+
+def test_all_users(mocker):
```

Comment:
It might be more intuitive to call this `test_get_user_by_id` or something. Also is there any reason to split this into two files? If we're grouping the queries themselves into one file it might be more intuitive to have the tests in one file as well (food for thought).